pkg/kncloudevents/event_dispatcher_test.go

```diff
@@ -944,9 +945,9 @@ func TestDispatchMessageToTLSEndpoint(t *testing.T) {
 	destinationEventsChan := make(chan cloudevents.Event, 10)
 	destinationReceivedEvents := make([]cloudevents.Event, 0, 10)
 	destinationHandler := eventingtlstesting.EventChannelHandler(destinationEventsChan)
-	destinationCA := eventingtlstesting.StartServer(ctx, t, 8334, destinationHandler, kncloudevents.WithDrainQuietPeriod(time.Millisecond))
+	destinationCA := eventingtlstesting.StartServer(ctx, t, 0, destinationHandler, kncloudevents.WithDrainQuietPeriod(time.Millisecond))
 	destination := duckv1.Addressable{
-		URL:     apis.HTTPS("localhost:8334"),
+		URL:     apis.HTTPS(fmt.Sprintf("localhost:%d", destinationCA.Port())),
```

Comment:
@satyampsoni `destinationCA` is a string value (containing the CA cert of the server) and does not have a `Port()` method.

Reply:
Hi @creydr 

Thank you for your feedback on the code review. I noticed the issue with using `Port()` on `destinationCA`. To address this, I'm planning to use a separate variable, let's call it `destinationPort`, to hold the dynamically assigned port.

Before proceeding, I wanted to confirm the availability of the `destinationPort` variable. Could you please let me know if such a variable is already present in the codebase or if there is a recommended method for obtaining the dynamically assigned port?

Your clarification on this matter will help me ensure that I'm addressing the issue correctly.

Reply:
> Before proceeding, I wanted to confirm the availability of the destinationPort variable. Could you please let me know if such a variable is already present in the codebase ...
Not sure, what you're trying to achieve :thinking: Can you give a bit more context here?

Reply:
Ah ok I think I understand you now. I am not aware of a similar var to `destinationPort`. Anyhow I would rather update the `StartServer()` to return the port too.